io/disk/vfc-tests/virtual_fc.py

```diff
@@ -310,44 +326,66 @@ def mpath_verification(self, operation, paths):
                        paths: %s" % (operation, paths))
 
         def is_path_online():
+            path_stat = []
+            process.system("multipathd -k'show paths'", ignore_status=True)
             path_stat = multipath.get_path_status(self.path)
-            if operation == "map":
+            self.log.info("operation:%s path=%s path_stat=%s \
+                           " % (operation, self.path, path_stat))
+            if operation in ('map', 'add'):
                 if path_stat[0] != "active" or path_stat[2] != "ready":
+                    self.log.info("opertion:%s path=%s stat=%s return=False \
+                                   " % (operation, self.path, path_stat))
                     return False
-                return True
-            elif operation == "unmap":
+                else:
+                    self.log.info("opertion:%s path=%s stat=%s return=True \
+                                   " % (operation, self.path, path_stat))
+                    return True
+            elif operation in ('unmap', 'remove'):
                 if path_stat[0] != "failed" or path_stat[2] != "faulty":
+                    self.log.info("opertion:%s path=%s stat=%s return=False \
+                                   " % (operation, self.path, path_stat))
                     return False
-                return True
+                else:
+                    self.log.info("operation:%s path=%s stat=%s return=True \
+                                   " % (operation, self.path, path_stat))
+                    return True
+            else:
+                self.log.info("Operation unknown, provide correct opertion")
 
         for path in paths:
             self.path = path
             if not wait.wait_for(is_path_online, timeout=10):
-                self.err_mesg.append("after %s path %s status did not \
-                                      changed" % (operation, path))
+                self.err_mesg.append("operation:%s path:%s verify failed \
```

Comment:
Can you make this format similar to other ones.
They are now `Op`, `op` and `operation`

Reply:
ok, will change it.